test/__snapshots__/test.js.snap

```diff
@@ -80,6 +80,155 @@ module.exports = {
 "
 `;
 
+exports[`Enum.webidl 1`] = `
+"\\"use strict\\";
+
+const conversions = require(\\"webidl-conversions\\");
+const utils = require(\\"./utils.js\\");
+
+const convertRequestDestination = require(\\"./RequestDestination.js\\").convert;
+const impl = utils.implSymbol;
+
+function Enum() {
+  throw new TypeError(\\"Illegal constructor\\");
+}
+
+Object.defineProperty(Enum, \\"prototype\\", {
+  value: Enum.prototype,
+  writable: false,
+  enumerable: false,
+  configurable: false
+});
+
+Enum.prototype.op = function op(destination) {
+  if (!this || !module.exports.is(this)) {
+    throw new TypeError(\\"Illegal invocation\\");
+  }
+
+  if (arguments.length < 1) {
+    throw new TypeError(
+      \\"Failed to execute 'op' on 'Enum': \\" + \\"1 argument required, but only \\" + arguments.length + \\" present.\\"
+    );
+  }
+
+  const args = [];
+  for (let i = 0; i < arguments.length && i < 1; ++i) {
+    args[i] = arguments[i];
+  }
+
+  args[0] = convertRequestDestination(args[0], { context: \\"Failed to execute 'op' on 'Enum': parameter 1\\" });
+
+  return this[impl].op(...args);
+};
+
+Object.defineProperty(Enum.prototype, \\"attr\\", {
+  get() {
+    if (!this || !module.exports.is(this)) {
+      throw new TypeError(\\"Illegal invocation\\");
+    }
+
+    return utils.tryWrapperForImpl(this[impl][\\"attr\\"]);
+  },
+
+  set(V) {
+    if (!this || !module.exports.is(this)) {
+      throw new TypeError(\\"Illegal invocation\\");
+    }
+
+    V = convertRequestDestination(V, { context: \\"Failed to set the 'attr' property on 'Enum': The provided value\\" });
```

Comment:
This isn't correct, per https://heycam.github.io/webidl/#dfn-attribute-setter step 3.6